ldap/servers/slapd/plugin_internal_op.c

```diff
@@ -282,12 +276,6 @@ slapi_search_internal_set_pb_ext(Slapi_PBlock *pb, Slapi_DN *sdn, int scope, con
         return;
     }
 
-    /* Free op/attrs just in case this pb is being reused */
```

Comment:
Not sure we should not keep these an rather add
 slapi_pblock_set(pb, SLAPI_OPERATION, NULL);
 slapi_pblock_get(pb, SLAPI_SEARCH_ATTRS, NULL);
 to avoid double free
 
 Same issue in two next changes